src/main/java/jalau/at18/azul/Wall.java

```diff
@@ -0,0 +1,32 @@
+package jalau.at18.azul;
+public class Wall {
+    private final int wallMatrixSize = 5;
+    private char[][] tiles = new char[wallMatrixSize][wallMatrixSize];
```

Comment:
Actually, I think you should use instances from enum `Tile` instead of char

Reply:
At the moment I am using char, later I will use tile since I must receive an arrayList from the Factory class.

Reply:
ok, if this is a work in progress then we are ok